contracts/protocol/clients/FermionFNFTPriceManager.sol

```diff
@@ -192,13 +187,23 @@ contract FermionFNFTPriceManager is FermionErrors, ERC2771Context, IFermionFNFTP
                 }
             }
         }
-
+        address msgSender = _msgSender();
         FermionTypes.FermionFractionsStorage storage fractionStorage = Common._getFermionFractionsStorage();
```

Comment:
Just noticed that you could also move this to the very beginning of the function and use `fractionStorage` in line 173 already.